skeleton/README.md

```diff
@@ -20,6 +20,23 @@ To build the project execute the following command:
 ```bash
   ./gradlew build
 ```
+#### Dependency Check
+
+The project is configured with the [OWASP Dependency Check gradle plugin](https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/index.html).
+This plugin checks for known vulnerabilities in the project's dependencies. To run the check, execute the following command:
+
+```bash
+  ./gradlew dependencyCheckAnalyze
+```
+
+It might take a while to check against the central vulnerability database (NVD) for the first time. The report will be generated in the `build/reports` directory.
+There are two ways to speed up the scan:
+
+- Use the local NVD database. The Platform Operations team maintains a [local copy of the NVD database](https://github.com/hmcts/cnp-owaspdependencycheck-update/tree/master).
+For more info how to configure the access to the local NVD database refer to the `GradleBuilder.groovy` file in the repository [Jenkins Shared Library](https://github.com/hmcts/cnp-jenkins-library)
```

Comment:
Maybe we should leave this part until after the tech improvement is discussed. Looks good otherwise